### ISSUE TITLE
Not working on inputs with different input types

### ISSUE THREAD

**Issue by desrosj** [neojp/gravity-forms-placeholders#2]
For some reason, when inputs have type="email" or type ="tel" the placeholder does not work.

**Comment by neojp** [comment]
Oh, I'm pretty sure I know why.

On line 13, on this file should have a conditional to include those input types.
https://github.com/neojp/gravity-forms-placeholders/blob/master/gf.placeholders.js.php#L13

.find('input[type=text], input[type=password], input[type=email], input[type=tel], textarea')

I can't edit and test this at the moment, in the meantime could you test it yourself?

**Comment by desrosj** [comment]
Nice! Those were the only two I tried. I know there are some more such as url, address, time. Maybe you could target all input/textarea types except type=submit?

**Comment by neojp** [comment]
I think I'll try this later tonight.

``` javascript
.find('input textarea').filter(function(i){
  var $field = $(this);
  if ($field.is('input')) {
    var type = $field.attr('type');
    return !(type == 'radio' || type == 'checkbox');
  }
  return true;
})
```

The problem I initially had, was that I'd apply the script to all input fields, including radio and checkboxes, making their labels dissapear, to filter against this 2 fields makes more sense I think.

Thanks for testing the plugin :)

**Comment by desrosj** [comment]
Cool. No rush, the plugin does what I need it to for now. But would be good for others and future uses. :)

## 

Jonathan Desrosiers
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by desrosj** [comment]
``` javascript
.find('input textarea').filter(function(i){
  var $field = $(this);
  if ($field.is('input')) {
    var type = $field.attr('type');
    return !(type == 'radio' || type == 'checkbox');
  }
  return true;
})
```

did not work. On my install I changed line 13 to

``` javascript
.find('input[type=text], input[type=email], input[type=url], input[type=tel], input[type=password], textarea')
```

Covers the fields I am using so far. I haven't done much with the other field types in HTML5 so I am unsure how placeholders would behave on mobile devices for field types such as date, datetime, month, and range.

**Comment by neojp** [comment]
I definitely have to take a good look at this, I want to support as many html input fields and custom gravity forms' fields as possible.

**Comment by neojp** [comment]
I just realized I missed a comma on the selector.

``` javascript
.find('input, textarea').filter(function(i){
```

**Comment by neojp** [comment]
Fixed on b2d685b589b5543660db89dbab75851cca9ab74e!  Sorry it took so long to make the commit, I also updated the SVN repo, so you can just download and update the plugin from there. Thanks for your help Jon.
